src/gql/graphql-client-api.tsx

```diff
@@ -2413,6 +2413,37 @@ export type AddInfoboxFieldMutation = (
   )> }
 );
 
+export type UpdatePropertyValueLatLngMutationVariables = Exact<{
+  propertyId: Scalars['ID'];
+  schemaItemId?: Maybe<Scalars['PropertySchemaFieldID']>;
+  itemId?: Maybe<Scalars['ID']>;
```

Comment:
Are these actually not necessary?

Reply:
This is the file generated automatically.